redisvl/llmcache/semantic.py

```diff
@@ -7,22 +7,22 @@
 from redisvl.llmcache.base import BaseLLMCache
 from redisvl.query import VectorQuery
 from redisvl.utils.utils import array_to_buffer
+from redisvl.schema import Schema
 from redisvl.vectorize.base import BaseVectorizer
 from redisvl.vectorize.text import HFTextVectorizer
 
 
 class SemanticCache(BaseLLMCache):
     """Semantic Cache for Large Language Models."""
 
-    # TODO: allow for user to change default fields
-    _vector_field_name: str = "prompt_vector"
-    _default_fields: List[Field] = [
-        VectorField(
-            _vector_field_name,
-            "FLAT",
-            {"DIM": 768, "TYPE": "FLOAT32", "DISTANCE_METRIC": "COSINE"},
-        ),
-    ]
+    _default_vector_field_name: str = "prompt_vector"
+    _default_vector_field: Dict[str, str] = {
```

Comment:
I think a default initialized ``Schema`` object makes sense here. Any user changes should use the user interface like we expect the user to when they manipulate the ``Schema`` object.

Reply:
I think we have to have schema update/manipulation if it's a user space object.